resources.html

```diff
@@ -24,7 +24,7 @@ <h3>Resources</h3>
 <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
 
 <script>
-    let HOST = "https://ctf-bot.herokuapp.com"
+    let HOST = "http://ctfbot.hats.sg"
```

Comment:
not https?

Reply:
@Enigmatrix @ahboon lol I have same question

Reply:
😔 time for actually learning certbot I guess. brb 1-2 weekends

Reply:
https://certbot.eff.org/ from experience it took me 5 mins just copy paste some commands